### ISSUE TITLE
Args test: use test table to write more tests

### ISSUE THREAD

**Issue by Thykof** [massalabs/as#63]
following https://github.com/massalabs/massa-as-sdk/issues/16 and the test table for envy test, we will be able to use test table to add more test and keep some demonstrative tests (with comments and how-to use guide).

This issue is blocked by https://github.com/massalabs/massa-as-sdk/issues/29

tasks:
- [ ] add more tests for Args class using test table transformer

**Comment by longgamma-massa** [comment]
@Thykof check this one pls and let me know the status atm

**Comment by Thykof** [comment]
status is TODO with low priority as it is adding more unit test to Args class
